pypdf/_writer.py

```diff
@@ -944,11 +958,17 @@ def update_page_form_field_values(
                 annotations and field data will be updated.
                 `List[Pageobject]` - provides list of pages to be processed.
                 `None` - all pages.
-            fields: a Python dictionary of field names (/T) and text
-                values (/V).
-            flags: An integer (0 to 7). The first bit sets ReadOnly, the
-                second bit sets Required, the third bit sets NoExport. See
-                PDF Reference Table 8.70 for details.
+            fields: a Python dictionary of:
+
+                * field names (/T) as keys and  text values (/V) as value
+                * field names (/T) as keys and  list of text values (/V) for multiple choice list
+                * field names (/T) as keys and  tuple of :
+                    * text values (/V)
+                    * font id (e.g. /F1, the font id must exist)
+                    * font size (0 for autosize)
+            flags: An integer. You can build it with InteractiveFormDictEntries:
+            ex: InteractiveFormDictEntries.Ff_ReadOnly ^ InteractiveFormDictEntries.Ff_Multiline
```

Comment:
The indentation seems to be off. Additionally, I recommend to link to the corresponding docs.

Reply:
re factored